src/utils/api.js

```diff
@@ -0,0 +1,5 @@
+export const getAllArticles = () => {
+  return fetch('https://jy-news.onrender.com/api/articles').then((response) =>
```

Comment:
Consider making a baseURL that you will be able to access, as if your api needed to change address, you would only need to do this in one area. Also, I would suggest using axios as it converts the data to json which is much easier for readability - but perhaps this one is fine as is! Here are the notes: https://notes.northcoders.com/courses/js-front-end/axios